### PR TITLE
Fleshed out readme and updated old dependency. 

### PR DESCRIPTION
**Pull request by GERey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,12 @@
...

-3. Run your application through the Emulator or on a real device.
\ No newline at end of file
+2. Copy the mobile key from your account settings page from your LaunchDarkly dashboard into `App.js`.
+3. Run `npm install start` in `hello-react-native`. This should download and create the directories for the project.
+4. Run `npm run start` to start the server to deliver the bundle url. Make sure this is running before you start your application. 
+
+##### iOS specific build instructions #####
+1. `cd` into `iOS` folder 
+2. run `pod update LaunchDarkly`
```
Comment (bwoskow-ld) [inline README.md]: Given that you're bumping the `LaunchDarkly` pod version below, I recommend that you do the pod install yourself and commit the changes. That way there is one less step for customers to have to run the sample app.

We generally shouldn't require customers to have to do a pod install. This was merely an omission on my part from when I bumped `hello-react-native` to v3 of the SDK.

**Review by bwoskow-ld — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (GERey) [reply README.md]: oh, I assumed we just didn't want to commit all of that to the repo. I'll do that and update.

Comment (GERey) [reply README.md]: I added the podfile and changed the readme accordingly. Let me know if there is anything else outstanding @bwoskow-ld .

File: README.md [hunk 2]
```diff
@@ -2,5 +2,9 @@
...
-2. Copy the mobile key from your account settings page from your LaunchDarkly dashboard into `App.js`

-3. Run your application through the Emulator or on a real device.
\ No newline at end of file
+2. Copy the mobile key from your account settings page from your LaunchDarkly dashboard into `App.js`.
+3. Run `npm install start` in `hello-react-native`. This should download and create the directories for the project.
+4. Run `npm run start` to start the server to deliver the bundle url. Make sure this is running before you start your application. 
+
+##### iOS specific build instructions #####
+4. Run your application through the Simulator or on a real device.
```
Comment (bwoskow-ld) [inline README.md]: Minor: this should now be # 1.

File: package.json [hunk 3]
```diff
@@ -9,7 +9,8 @@
   "dependencies": {
     "launchdarkly-react-native-client-sdk": "3.0.0",
     "react": "16.9.0",
-    "react-native": "0.61.2"
+    "react-native": "0.61.2",
+    "start": "^5.1.0"
```
Comment (bwoskow-ld) [inline package.json]: What is `start` used for?

Comment (GERey) [reply package.json]: that was my bad, I do npm install start, and it installs start. I need to change the read me again   
 -_-

**Review by bwoskow-ld — APPROVED** [review]

**Pull request merged**